packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -111,8 +111,8 @@ function Export({
           )}
           {error && (
             <Warning
-              isDanger
               isBelowInput
+              isDanger
```

Comment:
This is a change by the linter. Not sure why it appeared in my PR - prob no linter check in the pipeline in this repo? If so, that's bad - will cause more unwanted changes. Will take a look.

Reply:
Unlikely - it's the only modification that is applied when I run the linter for the entire repo.